test/integration/scheduler/workload_controller_test.go

```diff
@@ -295,7 +295,6 @@ var _ = ginkgo.Describe("Workload controller with scheduler", func() {
 				Obj()
 			gomega.Expect(k8sClient.Create(ctx, clusterQueue)).To(gomega.Succeed())
 			localQueue = testing.MakeLocalQueue("queue", ns.Name).ClusterQueue(clusterQueue.Name).Obj()
-			gomega.Expect(k8sClient.Create(ctx, localQueue)).To(gomega.Succeed())
```

Comment:
Leaving this case (Creating Lq before creating workload) might be worth it.
However, I think that we can work on follow-ups.

Reply:
I think we are fine like this. There are tests for both PodOverhead and LimitRanges that have the normal order.

Reply:
I see. If so, removing this case makes senese.